products/anolis8/transforms/table-srgmap.xslt

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cdf="http://checklists.nist.gov/xccdf/1.1" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+
+<xsl:include href="../../shared/transforms/shared_table-srgmap.xslt"/>
```

Comment:
The `table-srgmap.xslt` isn't used anywhere and also the `../../shared/transforms/shared_table-srgmap.xslt` that you included doesn't exist. It seems like you have copied something from an older version, but it's been removed. Please remove it.